### PR TITLE
Update: added _lang property for AAT multilang support (fixes #468)

### LINKED ISSUE #468: Add _lang property

### PR DESCRIPTION
**Pull request by chris-steele**
#468 

### Update
* Added support for multilang in AAT v1

### CODE REVIEW

**Comment by oliverfoster** [comment]
Should this go in both schema versions?

File: schema/content.schema.json [hunk 1]
```diff
@@ -25,6 +28,15 @@
         "editorOnly": true
       }
     },
+    "_lang": {
+      "type": "string",
+      "title": "Language",
+      "description": "The language/locale e.g. 'en' or 'fr-CA'.",
+      "default": "en",
+      "_backboneForms": {
+        "editorAttrs": { "disabled": true }
```
Comment (taylortom) [inline schema/content.schema.json:37]: You can also set `_backboneForms.showInUi` to `false` to hide the input completely (not sure which behaviour would be most appropriate).

Comment (chris-steele) [reply schema/content.schema.json:37]: good to know, yep possibly better hidden as lang menu shows currently selected language and user will never be able to edit this field

**Comment by chris-steele** [comment]
> Should this go in both schema versions?

@oliverfoster I don't think so as we're not backporting multilang to legacy

**Pull request closed without merging**